### DISCUSSION TITLE
bg-primary classes missing from action buttons?

### DISCUSSION THREAD

**Discussion by iancallahan** [filamentphp/filament#911] [General]
First of all, thank you for creating such a great product. I've used Filament to create quick admins for some side projects and it has worked beautifully. 

I'm using Filament 2 for the first time on an existing project and I have hit a snag that is probably because I've missed something. It seems that the bg-primary, etc.  classes are missing from action buttons. This leaves me with white text on a white background for all of the action buttons. 

I've set up a tailwind theme and that is working fine for the sidebar navigation, but the default action buttons for resources are all showing up this way. 

It looks like the second element in the  $buttonClasses array in the button component is not being applied, and I don't know why. 

<img width="338" alt="Screen Shot 2021-12-17 at 3 01 11 PM" src="https://user-images.githubusercontent.com/4192103/146601525-3fd5525a-1181-4c6d-9f09-721520476d75.png">
<img width="245" alt="Screen Shot 2021-12-17 at 3 01 01 PM" src="https://user-images.githubusercontent.com/4192103/146601526-bcee7954-4090-456e-8a7f-83953bf74464.png">

**Comment by danharrin** [comment]
Hey, I've not seen this issue before. Can I see your `tailwind.config.js` please?

**Comment by danharrin** [comment]
I've created an issue for this in #913, please reply on there.
